srttools/monitor/webserver.py

```diff
@@ -263,15 +82,32 @@ def get(self):
         self.write(open("index.html").read())
 
 
+class FaviconHandler(RequestHandler):
+    def initialize(self):
+        # Load the favicon in memory
+        favicon_data = None
+        with files("srttools.monitor").joinpath("resources", "favicon.ico").open("rb") as f:
+            favicon_data = f.read()
+        self.favicon_data = favicon_data
+
+    def get(self):
+        self.set_header("Content-Type", "image/x-icon")
+        self.write(self.favicon_data)
```

Comment:
The favicon for the webpage is basically [this](https://github.com/discos/srt-single-dish-tools/blob/main/docs/images/srttools.ico). I simply replaced the black background with a transparent one.